mkdocs.yml

```diff
@@ -356,7 +357,6 @@ nav:
               - Multi-GPU Training: yolov5/tutorials/multi_gpu_training.md
               - PyTorch Hub: yolov5/tutorials/pytorch_hub_model_loading.md
               - TFLite, ONNX, CoreML, TensorRT Export: yolov5/tutorials/model_export.md
-              - NVIDIA Jetson Nano Deployment: yolov5/tutorials/running_on_jetson_nano.md
```

Comment:
@lakshanthad if we delete a URL we need to add a redirect in mkdocs.yml later down around L700, otherwise this URL will be now and forever reported as 404 by Google and all the other search engine consoles, negatively impacting site SEO.

Reply:
@glenn-jocher Thank you for pointing it out! I have fixed it. Also, I will keep this in mind in the future :)